installer.sh

```diff
@@ -16,7 +18,7 @@ echo "password when required."
 echo
 echo
 
-if [ "$(id -u)" -eq 0 ] ; then
+if [ "$(id -u)" -eq 0 ] ; then #Check for root access
```

Comment:
Please add a the comment in the line above. Although it is not really needed as the code is self-explaining.